tests/CompilerTest/Model/CoordinateTest.cs

```diff
@@ -29,5 +29,23 @@ public void TestItRepresentsAsString()
         {
             Assert.Equal("abc def", coordinate.ToString());
         }
+
+        [Fact]
+        public void TestDegreeMinSecToDecimalDegree() {

```

Comment:
Could we make this test and the one below a `[Theory]`? This will mean that we can see if individual test cases fail, rather than the whole thing :)

Reply:
Yep :)